### ISSUE TITLE
moveit: mico_moveit_config setup_assistant.launch

### ISSUE THREAD

**Issue by gsubramani** [RIVeR-Lab/wpi_jaco#43]
Hi,
Fist off, thanks for building the libraries! I am having trouble using moveit! with the wpi libraries. I can see that the setup_assistant.launch file isn't working properly. It needs .setup_assistant to work. Maybe the file is missing? 
Thank you!
Guru

**Comment by dliu62** [comment]
Having the same issue here. Was trying to config moveit to add a command group for the gripper, but without the .setup_assistant file the moveit setup wizard cannot be used. 

Also, creating a new config using the urdf.xarco in mico_description folder gives me errors: [ERROR] [1468357948.135032491]: No name given for the robot.
If i then add a name to the robot in the URDF file, it gives me another error: [ERROR] [1468358030.202707718]: No link elements found in urdf file.

 Any idea why this is happening?

**Comment by dekent** [comment]
Yeah, the .setup_assistant file is missing.  My lab didn't create this one (and we don't have a mico arm at the moment), so I assume that file is lost.  The best solution would be to create a new MoveIt! config.

Are you getting that error when creating the config using the urdf in mico_description/robots/standalone_arm.urdf.xacro or in mico_description/urdf/mico_arm.urdf.xacro?  You should use the file in the robots directory.
